TradeItIosTicketSDK2/TradeItYahooTradingTicketViewController.swift

```diff
@@ -66,51 +65,7 @@ class TradeItYahooTradingTicketViewController: TradeItYahooViewController, UITab
             return
         }
 
-        // Check if selected account supports trading equities
-        TradeItSDK.linkedBrokerManager.getAvailableBrokers(
-            onSuccess: { brokers in
-                guard let broker = brokers.first(
-                    where: { broker in
-                        return broker.shortName == self.order.linkedBrokerAccount?.linkedBroker?.brokerName
-                    }
-                ), broker.equityServices()?.supportsTrading == true else {
-                    self.alertManager.showAlertWithMessageOnly(
-                        onViewController: self,
-                        withTitle: "Unsupported Account",
-                        withMessage: "The selected account does not support trading stocks. Please choose another account.",
-                        withActionTitle: "OK",
-                        onAlertActionTapped: {
-                            self.delegate?.invalidAccountSelected(
-                                onTradingTicketViewController: self,
-                                withOrder: self.order
-                            )
-                        }
-                    )
-                    return
-                }
-            },
-            onFailure: { _ in
-                self.alertManager.showAlertWithMessageOnly(
-                    onViewController: self,
-                    withTitle: "Error",
-                    withMessage: "Could not determine if this account can trade stocks. Please try again.",
-                    withActionTitle: "OK",
-                    onAlertActionTapped: {
-                        self.delegate?.invalidAccountSelected(
-                            onTradingTicketViewController: self,
-                            withOrder: self.order
-                        )
-                    }
-                )
-            }
-        )
-
-        self.reloadTicket()
-    }
-
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
-        self.fireViewEventNotification(view: .trading, title: self.title)
+        self.updateOrderCapabilities()
```

Comment:
I thought we were going to separate this stuff so we don't fetch balances every time we come back to the screen...

Reply:
I haven't updated yet the Yahoo ticket. I wanted to confirm that was correct for the Trade it Ticket to update it.